### DISCUSSION TITLE
About Radio Component

### DISCUSSION THREAD

**Discussion by WAELKASSEM** [Megabit/Blazorise#1626] [Q&A]
**Hello**,
I am using multiple [Radio Components](https://blazorise.com/docs/components/radio/) **without** the Radio Group.

But I can't get the **bind-Checked** attribute to work correctly...**It only works on the first click.** True doesn't revert back to false in code. However in terms of UI it works fine.  I've tried to trigger StateHasChanged() and multiple other approaches but no luck.

I've used this syntax : 

`<Radio TValue="bool" Group="choices"  Value="@true" @bind-Checked="choice1"></Radio>`

I'm using the @onclick event to change the boolean values manually... but for multiple radio buttons it's an issue.

Is this a bug? am I missing something ?

**Comment by stsrki** [comment]
It's probably because you have `Value="@true"`. Try removing it. Or use it like `Value="@choice1"`.

**Reply by WAELKASSEM** [reply]
`<Radio TValue="bool" Group="choices" @bind-Checked="choice1"></Radio>`
Tried this, it didn't work. 
Even tried the @ref without the @bind-Checked, to get the checked value. No luck either.
